save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/HistoryView.kt

```diff
@@ -42,36 +50,45 @@ class HistoryView : RComponent<HistoryProps, RState>() {
                         +"${it.row.index}"
                     }
                 }
-                column("date", "Date") {
+                column("status", "Status") {
                     td {
-                        +"${it.value["date"]}"
+                        +"${it.value.status}"
                     }
                 }
                 column("passed", "Passed") {
                     td {
-                        +"${it.value["passed"]}"
+                        +"${it.value.passedTests}"
                     }
                 }
                 column("failed", "Failed") {
                     td {
-                        +"${it.value["failed"]}"
+                        +"${it.value.failedTests}"
                     }
                 }
                 column("skipped", "Skipped") {
                     td {
-                        +"${it.value["skipped"]}"
+                        +"${it.value.skippedTests}"
                     }
                 }
             }
         ) { _, _ ->
-            // todo: fetch data from backend using `window.location.origin`
-            arrayOf(
-                json("date" to "26-Jan-2016", "passed" to "26", "failed" to "17", "skipped" to "36"),
-                json("date" to "76-Jun-2019", "passed" to "67", "failed" to "75", "skipped" to "236"),
+
+            post(
+                url = "${window.location.origin}/executionDtoByProject",
+                headers = Headers().also {
+                    it.set("Accept", "application/json")
+                    it.set("Content-Type", "application/json")
+                },
+                body = JSON.stringify(getProject(props.name, props.owner))
```

Comment:
Can we make `executionDtoByProject` accept name and owner? This way we will make one query to backend instead of two, and backend can query the whole project on its own